### ISSUE TITLE
dunfell: Creating a dunfell-clang14 branch

### ISSUE THREAD

**Issue by rakuco** [kraj/meta-clang#835]
This has been discussed in the meta-browser repository for a while, most recently in OSSystems/meta-browser#734. Clang 12 is too old for Chromium at this point as the project continues to adopt more C++20 features, and the lack of support for building the native bits with meta-clang's libc++ causes the build to fail either because the host's libstdc++ is too old (OSSystems/meta-browser#740) or just incompatible with the rest of the build (OSSystems/meta-browser#726).

A few weeks ago I created a dunfell-clang14 branch in my meta-clang fork (https://github.com/rakuco/meta-clang/tree/dunfell-clang14) that contains a lot of cherry-picks from meta-clang's kirkstone branch on top of a dunfell-clang12 branch. This tree has been tested with meta-browser, and chromium-x11 and chromium-ozone-wayland build and run fine on different architectures.

What would be the procedure for getting this work reviewed and integrated into this repository? I'm pretty sure I've cherry-picked some unneeded changes, and skipped several others that were not relevant to meta-browser (like all the bpf, bcc and android-tools commits).

**Comment by nrpt-m** [comment]
@kraj,
It's been pending from last 3 weeks. Could you please let @rakuco know about the procedure for getting this work reviewed and integrated into this repository?

**Comment by rwmacleod** [comment]
@rakuco seems that @kraj is waiting for a PR. Do you have time for that soon?



On Thu, Sep 7, 2023 at 4:16 AM Randy MacLeod <rwmacleod@gmail.com> wrote:
>
> Have a great vacation! Thanks for letting us know that you'll be away.
>
> Khem,
>
> Is meta-clang/dunfell-clang14 waiting on you having some time to work on it or is it blocked by something else?

I think Rakuco has already done the work, so I have asked him to send
a pull against this branch. He has not yet done it.

Hide quoted text



>
> Randy
>
>

**Comment by rakuco** [comment]
> I think Rakuco has already done the work, so I have asked him to send
> a pull against this branch. He has not yet done it.

#853 finally sent!

**Comment by rakuco** [comment]
Hmm, dunno why this issue did not get closed when #853 was merged. The new branch is in, and it corresponds to what I had in my fork.
